src/state/actions/index.js

```diff
@@ -32,6 +39,7 @@ export const getDocs = authState => async dispatch => {
       dispatch({ type: SET_DOCS, payload: Response });
     } else {
       dispatch({ type: FINISH_FETCH });
+      dispatch({ type: SEARCH_BAR });
     }
   } catch (err) {
     console.log(err);
```

Comment:
Should this be just a console.error or were you planning on making this into something that plugs into a display of the error for the user?

Reply:
Please someone who actually worked on this correct me if I'm wrong, but I believe that's what they're using to get the no search results pop up.

Reply:
Myself or a UX will be adding a Ant Design model in replace of the alert for better looking user experience tomorrow 12-15